misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/picking/PickAlternatives.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import PickAltStepButton from './PickAltStepButton';
+class PickAlternatives extends Component {
+  renderAlternatives = () => {
+    // const { pickFromAlternativeIds, pickFromAlternatives, altSteps } = this.props;
+
+    const { genSteps, wfProcessId, activityId, lineId, stepId } = this.props;
+
+    // let totalQty = 0;
+
+    // const alternativeHUs = pickFromAlternatives.reduce((accumulator, item) => {
+    //   if (pickFromAlternativeIds.includes(item.id) && totalQty < diffToPick && diffToPick) {
+    //     accumulator.push(item);
+    //     totalQty += item.qtyAvailable;
+    //   }
+    //   return accumulator;
+    // }, []);
+
+    // console.log('DIFFtoPick:', diffToPick);
+    // console.log('ALTHUs', alternativeHUs);
+
+    // calculate here - filter to achieve the amount
+    return (
+      <div>
+        {Object.keys(genSteps).length > 0 &&
+          Object.keys(genSteps).map((altStepKey) => {
+            return (
+              <PickAltStepButton
+                key={genSteps[altStepKey].id}
+                wfProcessId={wfProcessId}
+                activityId={activityId}
+                lineId={lineId}
+                stepId={stepId}
+                altStepId={genSteps[altStepKey].id}
+                {...genSteps[altStepKey]}
```

Comment:
Pls avoid spreading props on react components. It's not a good practice.

Reply:
Why to not used? it's nicely explained here:
* https://codeburst.io/react-anti-pattern-jsx-spread-attributes-59d1dd53677f
* https://aurelio.me/blog/Why-I-stopped-spreading-props-react/

Reply:
Done